<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionDialog</class>
 <widget class="QDialog" name="ConnectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>200</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Connection Menu</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="lo_main">
     <item>
      <layout class="QGridLayout" name="lo_input">
       <item row="3" column="0">
        <widget class="QLabel" name="lb_login">
         <property name="text">
          <string>Login</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lb_port">
         <property name="text">
          <string>Port</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lb_password">
         <property name="text">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lb_host">
         <property name="text">
          <string>Host</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lb_databaseName">
         <property name="text">
          <string>Database Name</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="le_host"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="le_database_name"/>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="le_login"/>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="le_password">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="sb_port">
         <property name="contextMenuPolicy">
          <enum>Qt::NoContextMenu</enum>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="maximum">
          <number>9999</number>
         </property>
         <property name="stepType">
          <enum>QAbstractSpinBox::DefaultStepType</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="lo_buttons">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item>
        <widget class="QPushButton" name="pb_ok">
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_cancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pb_cancel</sender>
   <signal>clicked()</signal>
   <receiver>le_host</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_cancel</sender>
   <signal>clicked()</signal>
   <receiver>sb_port</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_cancel</sender>
   <signal>clicked()</signal>
   <receiver>le_database_name</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_cancel</sender>
   <signal>clicked()</signal>
   <receiver>le_login</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_cancel</sender>
   <signal>clicked()</signal>
   <receiver>le_password</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
